Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM node:12.13.0-alpine AS node
+WORKDIR /app
+
+
+
+#
+# Stage: Production NPM install
+#
+FROM node AS npm-prod
+
+COPY package.json \
+    package-lock.json \
+    ./
+
+RUN npm install --production
+
+
+
+#
+# Stage: Production environment
+#
+FROM node AS prod
+ENV NODE_ENV=production
+
+COPY --from=npm-prod /app/ .
+
+CMD ["sleep", "86400"]
```

Comment:
does `sleep infinity` work in Alpine? Have seen this before

Reply:
Nope: https://github.com/libero/storybook/pull/2#discussion_r328218038